### ISSUE TITLE
🐛 BUG: Extension duplicates color symbols when using with Tailwind Intellisense

### ISSUE THREAD

**Issue by czernika** [withastro/language-tools#437]
### Describe the Bug

VSCode: with enabled [TailwindCSS Intellisense](https://marketplace.visualstudio.com/items?itemName=bradlc.vscode-tailwindcss) it duplicates color symbols in `.astro` and class hover for `.vue` (at least `.vue`, see screenshot)

![Снимок экрана от 2022-10-21 17-34-05](https://user-images.githubusercontent.com/86893348/197224645-e2f1b63d-0951-47fc-ae5d-6913c1a4b863.png)

If you disable Tailwind extension it shows only one square as expected

### Steps to Reproduce

1. Start [new](https://docs.astro.build/en/install/auto/#1-run-the-setup-wizard) project and [add](https://docs.astro.build/en/guides/integrations-guide/tailwind/#quick-install) support for TailwindCSS
```sh
npm create astro@latest
npx astro add tailwind
```
2. Add some color-class to any element (eg `bg-red-500`)
3. It duplicates color symbol square as it showed in screenshot in `.astro`

![Снимок экрана от 2022-10-21 17-34-20](https://user-images.githubusercontent.com/86893348/197225125-9ae19e35-cbf3-4735-8c32-1c446dbf6514.png)

**Comment by Princesseuh** [comment]
Hello, thank you for submitting this issue!

I am not able to reproduce this in `.astro` files (and our extension doesn't run in `.vue` files in any way). Here's how things looks for me: 
<img width="650" alt="image" src="https://user-images.githubusercontent.com/3019731/197234688-4a42fdc6-6e97-4080-9635-2730f89ac367.png">

Additionally, our extension does not provide color symbols for Tailwind classes (or for classes in general, actually). So I don't think it's on our end?

**Comment by czernika** [comment]
@Princesseuh Yeah, I guess if it is cannot be reproduced the problem somewhere on my side. At least I know I should look somewhere else, thanks)

**Comment by czernika** [comment]
Oh, I've found it! It was [WindiCSS](https://marketplace.visualstudio.com/items?itemName=voorjaar.windicss-intellisense) extension I forgot about

I'm usually working with Vue+WindCSS or Laravel/PHP+TailwindCSS, so I have few separate workspaces with a list of extensions for each workspace (Windi disabled on Laravel workspace, while Tailwind on Vue). 

I've started a new Astro project for the first time so there was no workspace and both extensions were enabled - that's why color symbol was shown twice and even with Tailwind extension disabled and that's why I thought it was Astro extension. I completely forgot WindiCSS extension even exists in my workspace, sorry for that

Astro is great! Thanks for your work and help
